### ISSUE TITLE
RemoteFileDirtyPageThreshold key value

### ISSUE THREAD

**Issue by DJ8014** [MicrosoftDocs/windowsserverdocs#7643]
"This threshold can be controlled with the following regkey: HKLM\SYSTEM\CurrentControlSet\Control\Session Manager\Memory Management\RemoteFileDirtyPageThreshold"

How should this be entered? As a value in the (Default) REG_SZ field?

![image](https://github.com/MicrosoftDocs/windowsserverdocs/assets/4950759/3d0cae32-f481-4388-a9df-380beb794b6e)

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 6fe81add-b285-ff8e-f383-2df80332e722
* Version Independent ID: 4e96ec10-8088-0d77-bc8b-d2197aebf9d9
* Content: [Troubleshoot Cache and Memory Manager Performance Issues](https://learn.microsoft.com/en-us/windows-server/administration/performance-tuning/subsystem/cache-memory-management/troubleshoot#remote-file-dirty-page-threshold-is-consistently-exceeded)
* Content Source: [WindowsServerDocs/administration/performance-tuning/subsystem/cache-memory-management/troubleshoot.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/main/WindowsServerDocs/administration/performance-tuning/subsystem/cache-memory-management/troubleshoot.md)
* Product: **windows-server**
* Technology: **administration**
* GitHub Login: @phstee
* Microsoft Alias: **pavel**

**Comment by DJ8014** [comment]
Tagging @Xelu86 in hopes of a response. Any assistance is appreciated.

**Comment by Xelu86** [comment]
Pending clarity from MM team surrounding RemoteFileDirtyPageThreshold registry entry for documentation update.

**Comment by DJ8014** [comment]
Thanks. I think I know that the DWORD value is the proper way to enter it. The greater issue is that it is not having any effect on Windows Server 2022.

________________________________

**Comment by Xelu86** [comment]
Correct, the reg key type should be a DWORD (which was not made clear) with the value being dependent upon different factors. We waiting to hear back if this entry functions as stated in WS2016/WS2019 and if anything has changed in WS2022. We'll update this document as needed once this information is investigated.

**Comment by DJ8014** [comment]
Thanks. Would be great to get this fixed.

**Comment by DJ8014** [comment]
I see that the page has been updated to clarify the proper settings (DWORD, etc.). Any word on whether this registry setting should be working in WS2022, and if not, if a fix may be forthcoming? Thanks!

**Comment by Xelu86** [comment]
Hey @DJ8014, still waiting on further updates to clarify this.

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
